regression/cbmc-java/exceptions24/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+test.class
+--java-throw-runtime-exceptions
+^EXIT=10$
+^SIGNAL=0$
+VERIFICATION FAILED
```

Comment:
We should have some positive tests as well, to show that function that would not throw exception are detected by CBMC as such.